Description: fix-anjuta-doc-macro
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 guimup (0.3.3-debian1) wheezy; urgency=low
 .
   * Initial release
   * Switch to dpkg-source 3.0 (quilt) format
Author: Ronald van Engelen <rvengelen@hethooghuis.nl>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- guimup-0.3.3.orig/Makefile.am
+++ guimup-0.3.3/Makefile.am
@@ -3,8 +3,8 @@
 
 SUBDIRS = src 
 
-guimupdocdir = ${prefix}/doc/guimup
-guimupdoc_DATA = \
+#guimupdocdir = ${prefix}/doc/guimup
+#guimupdoc_DATA = \
 	README\
 	COPYING\
 	AUTHORS\
@@ -13,9 +13,9 @@ guimupdoc_DATA = \
 	NEWS
 
 
-EXTRA_DIST = $(guimupdoc_DATA)
+#EXTRA_DIST = $(guimupdoc_DATA)
 
 
 # Remove doc directory on uninstall
-uninstall-local:
-	-rm -r $(guimupdocdir)
+#uninstall-local:
+#	-rm -r $(guimupdocdir)
